lib/card_generator.rb

```diff
@@ -0,0 +1,62 @@
+require './lib/card'
+require './lib/deck'
+require './lib/player'
+require './lib/turn'
+require './lib/game'
+
+require 'pry'
+
+class CardGenerator
+
+    attr_reader :deck1,
+                :deck2
+
+    def initialize(file_name)
```

Comment:
This looks great! I do think this class is doing too much though. I'd have stopped around line 37 after creating all the cards. Then, your `Game` can still be responsible for shuffling and divvying out the cards. The `CardGenerator`'s only job should be generating cards. I'd also like to have seen tests for this class.